### ISSUE TITLE
Blackscreen on Final Fantasy Fables: Chocobo Tales

### ISSUE THREAD

**Issue by theGIZOID** [AdmiralCurtiss/WfcPatcher#11]
Like the title itself already says, the game "Final Fantasy Fables: Chocobo Tales" cannot be patched.
After patching the game just leaves a blackscreen on start up.

**Comment by AdmiralCurtiss** [comment]
Cannot confirm, works fine using the European version. Make sure you're using a proper dump, with an MD5 checksum of a392490af90e31f2ce68d18b11833080.

**Comment by theGIZOID** [comment]
Well thats odd. I have the same checksum and it still does not work. I was using WfcPatcher 1.5 Should I try an older version?

P.S. After patching it it has a MD5 checksum of "3642bbc84837926e1d92ec07f7685cc1"
Is this the correct checksum?

**Comment by AdmiralCurtiss** [comment]
Yeah I'm getting the same checksum after patching. Probably a flashcard issue, maybe look up if the game uses some anti-piracy code and if there's a manual patch available instead of relying on the flashcard to automatically detect that, I suspect patching the game's executables could mess that up.

**Comment by theGIZOID** [comment]
Well the unpatched rom works fine on my card as well as the cheat code using WfcReplay 0.5 but I still preffer a patched rom (I don't want to mess around with the cheat database) Also the game does not have an AP patch whatsoever.

**Comment by the4anoni** [comment]
Try desmume

**Comment by theGIZOID** [comment]
No, thats not the issue. While the patched image works perfectly fine on Desmume it does not on my flashcard (and I want to have it working on my flashcard). But there isn't anything I can do about that. In fact: every single game made by Square Enix does no longer work on my flashcard after patching it for some reason. At least cheat codes do work so I just save the codes into a .txt file as a backup just in case I want to revert my cheat database.

**Comment by the4anoni** [comment]
Which flashcard you have ?

**Comment by theGIZOID** [comment]
"R4i RTS Gold 3DS Revolution" running on the latest upgrade patch and Wood R4 kernel. I don't request a specific fix for that as I am still fine with the cheat codes as long as I keep the WfcReplay codes in a seperate .txt file, but of course I'd be nice to see the patched version working on my flashcard as well.

**Comment by the4anoni** [comment]
On r4i dual core 2015 also these same issues ,but nobody make other Fw so buy dstwo or ak2i (ak2i works ok )... Ak 2 i works on dsi 1.4.4 or 3ds 4.3 (?) ,but if you have cfw on 3ds (not i sadly) you can use it on 11.0

**Comment by theGIZOID** [comment]
I neither have a DSi nor a 3DS, only my regular DS Lite. I have no interests in buying a second and more expensive flashcard just because of a few games not working after patching. Like I said: I am totally fine with using codes. Besides: My card works on both other systems on their latest firmware, I have tested this by myself but thanks for the tip anyways.

**Comment by the4anoni** [comment]
Buy Ak 2i. It is very cheap :)
